snap/info.go

```diff
@@ -94,6 +94,9 @@ type PlaceInfo interface {
 	// XdgRuntimeDirs returns a glob that matches all XDG_RUNTIME_DIR
 	// directories for all users of the snap.
 	XdgRuntimeDirs() string
+
+	// UserExposedHomeDir returns the snap's new home directory under ~/Snap.
+	UserExposedHomeDir(home string) string
```

Comment:
Is this used in this PR?

Reply:
No, it's just something I should have added before. It's used in the next snapshot PR.